packages/babel-parser/test/fixtures/es2015/duplicate-proto/without-assignment-expression/input.js

```diff
@@ -0,0 +1,4 @@
+({
+  __proto__: a,
+  __proto__: a,
+})
```

Comment:
We already have a similar test in `packages/babel-parser/test/fixtures/es2015/uncategorised/348/input.js`, can you remove that uncategorised tests?

Reply:
Okay, I will remove it.